auth-api/src/auth_api/resources/user.py

```diff
@@ -218,3 +218,33 @@ def get():
         """Add a new contact for the Entity identified by the provided id."""
         sub = g.jwt_oidc_token_info.get('sub', None)
         return AuthorizationService.get_user_authorizations(sub), http_status.HTTP_200_OK
+
+
+@cors_preflight('PATCH, OPTIONS')
+@API.route('/termsofuse', methods=['PATCH', 'OPTIONS'])
```

Comment:
This can be a patch on user resource (not a termsofuse endpoint), as it's basically patching the user resource with terms of use details.

Reply:
Agreed. writing a decent patch method supporting the patch for a user is tough..Thats why went for this approach , though bad in terms of rest principles..will change this with a TODO
something like below is ideal , but havent see any

[
{ “op”: “update”, “path”: “/termsflag”, “value”: true }
]